airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_factory/DefaultSyncJobFactory.java

```diff
@@ -65,15 +69,24 @@ public Long create(final UUID connectionId) {
         standardSyncOperations.add(standardSyncOperation);
       }
 
+      ActorDefinitionResourceRequirements sourceResourceRequirements = sourceDefinition.getResourceRequirements();
+      ActorDefinitionResourceRequirements destinationResourceRequirements = destinationDefinition.getResourceRequirements();
+
+      // for OSS users, make it possible to ignore default actor-level resource requirements
+      if (!connectorSpecificResourceDefaultsEnabled) {
+        sourceResourceRequirements = null;
+        destinationResourceRequirements = null;
+      }
```

Comment:
This logic could be moved into the `JobCreator.createSyncJob()` method so that it doesn't need to be repeated